src/server/components/proxyMiddleware/handlers.ts

```diff
@@ -0,0 +1,83 @@
+import type { Request, Response } from "express";
+import { validateCountryLower } from "server/models/listItem/providers/helpers";
+import { listExists, serviceTypeSchema } from "server/components/proxyMiddleware/helpers";
+import { countriesList } from "server/services/metadata";
+import { camelCase } from "lodash";
+
+export const handlers = {
+  start: {
+    get: getStartPage,
+  },
+  countrySelect: {
+    get: getCountrySelectPage,
+    post: postCountrySelectPage,
+  },
+  stop: {
+    get: getStopPage,
+  },
+};
+
+function getStartPage(req: Request, res: Response) {
```

Comment:
I won't block review on this point but something to keep in mind:

I'd split the handlers below into their own files, grouped by route. There is less likely to be a merge conflict if two people are working on separate apply router handlers, and then you can just take advantage of modules.

if you remember dashboard/controllers.ts, the file was extremely long (almost 1k lines) which makes it makes it annoying to navigate and impossible to move stuff.
 
```
// handlers/index.ts 
export { * as start } from "./start"
export { * as countrySelect } from "./countrySelect"
```

```
// applyRouter.ts
import { * as handlers } from "./handlers"; // if no file is specified, index file is used.

handlers.start.get
handlers.countrySelect.get 
```